cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -278,7 +280,7 @@ func (p *BicepProvider) Destroy(
 	return async.RunInteractiveTaskWithProgress(
 		func(asyncContext *async.InteractiveTaskContextWithProgress[*DestroyResult, *DestroyProgress]) {
 			asyncContext.SetProgress(&DestroyProgress{Message: "Fetching resource groups", Timestamp: time.Now()})
-			rgsFromDeployment, err := p.getResourceGroupsFromDeployment(ctx)
+			rgsFromDeployment, err := p.getResourceGroups(ctx)
```

Comment:
This may be a result of the merge conflict. I do prefer a naming that clearly shows where the source of truth is coming from.

The story here is that knowing whether the resource groups are live resource groups that we queried for vs. possibly stale ones due to discovery from an old deployment object is maybe important. I think it calls out more clearly on whether checking resource group existence states should be necessary.

I'm fine with either `getResourceGroupsFromDeployment` or `getResourceGroupsFromLatestDeployment`.

Reply:
> This may be a result of the merge conflict. I do prefer a naming that clearly shows where the source of truth is coming from.

100% agree.  The old name is much better.  I'll change this to be something more descriptive.